### DISCUSSION TITLE
mac下使用命令行编译失败，错误是没有字体

### DISCUSSION THREAD

**Discussion by wanpiqiu123** [sjtug/SJTUThesis#598] [提问] [answered]
**问题描述**:
mac下使用命令行编译失败，错误是没有字体: Package fontspec Error: The font "Kaiti SC" cannot be found.
错误来源应该是ctex，在这里定义的:/Users/leo/Library/Application Support/MiKTeX/texmfs/install/tex/latex/ctex/fo
ntset/ctex-fontset-macnew.def
看了一下mac的字体库好像没有Kaiti SC

**系统信息**:

- OS版本(带版本号): macOS 10.15.1
- TeX 发行版(带版本号): MiKTeX 21.3-darwin-x86_64

**日志信息**:

https://pastebin.ubuntu.com/p/JFQPG79cvY/

**Comment by skyzh** [comment, marked as answer]
https://support.apple.com/zh-hk/HT210192 Kaiti SC 需要自行下载。请在 Font Book 应用中操作。

**Reply by lengyijun** [reply]
manjaro 下怎么办？

**Reply by lengyijun** [reply]
有没有谁能给一份 linux 下 `fc-list` 的输出，我好比较一下缺少了什么字体

**Reply by skyzh** [reply]
Arch Linux 系上确实比较麻烦，我之前也没有成功一次装完。有两种办法：
* 把 https://wiki.archlinux.org/title/Fonts_(%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87) 里面的中文字体全部装一遍，然后手动把 LaTeX 里面的一些自带字体链接进 fc-list，印象里不是很多。
* 安装 Adobe 字体并指定 `fontset=adobe`.

**Reply by lengyijun** [reply]
我平平无奇的解决方案：
1. 卸载pacman里所有的texlive, 改用texlive.iso安装
2. 删除 `/usr/share/fonts/` 里手工复制进去的字体文件

**Reply by skyzh** [reply]
确实可以 🤣🤣🤣
